assets/CSS/styles.css

```diff
@@ -179,14 +187,19 @@ img {
 
 aside {
   position: fixed;
-  height: 801px;
+  top: 0;
+  height: 100%;
   overflow-y: scroll;
   width: 210px;
   padding: 12px;
   flex-shrink: 0;
   padding-right: 19px;
 }
```

Comment:
Great job so far guys, I suggest putting the native HTML tag at first, it is more consistence, this is not a blocker ok... just a suggest

Reply:
Ok, i'll keep it in mind thanks!